classes/task/PKPUsageStatsLoader.php

```diff
@@ -114,88 +105,31 @@ protected function executeActions(): bool
             $this->autoStage();
         }
         $processFilesResult = parent::executeActions();
-        foreach ($this->months as $month) {
-            dispatch(new CompileMonthlyMetrics($month, Application::get()->getRequest()->getSite()));
-        }
-        return ($processFilesResult && !$processingDirError);
-    }
-
-    /**
-     * Delete entries in usage stats temporary tables by loadId
-     */
-    protected function deleteByLoadId(string $loadId): void
-    {
-        $this->temporaryInstitutionsDao->deleteByLoadId($loadId);
-        $this->temporaryTotalsDao->deleteByLoadId($loadId);
-        $this->temporaryItemInvestigationsDao->deleteByLoadId($loadId);
-        $this->temporaryItemRequestsDao->deleteByLoadId($loadId);
-    }
 
-    /**
-     * Insert usage stats log entry into temporary tables
-     */
-    protected function insertTemporaryUsageStatsData(object $entry, int $lineNumber, string $loadId): void
-    {
-        try {
-            $this->temporaryTotalsDao->insert($entry, $lineNumber, $loadId);
-            $this->temporaryInstitutionsDao->insert($entry->institutionIds, $lineNumber, $loadId);
-            if (!empty($entry->submissionId)) {
-                $this->temporaryItemInvestigationsDao->insert($entry, $lineNumber, $loadId);
-                if ($entry->assocType == Application::ASSOC_TYPE_SUBMISSION_FILE) {
-                    $this->temporaryItemRequestsDao->insert($entry, $lineNumber, $loadId);
-                }
+        if ($processFilesResult) {
```

Comment:
I think it's better to use early returns when possible, to decrease the code indentation and make it easier to read.
So here, I'd change to:
```php
if (!$processFilesResult) {
    return false;
}
```